AdyenComponents/Apple Pay/ApplePayConfiguration.swift

```diff
@@ -100,7 +126,15 @@ extension ApplePayComponent {
                 paymentRequest.supportsCouponCode = supportsCouponCode
                 paymentRequest.shippingContactEditingMode = allowShippingContactEditing ? .enabled : .storePickup
             }
+            
+            return paymentRequest
+        }
 
+        internal mutating func paymentRequest(with supportedNetworks: [PKPaymentNetwork]) -> PKPaymentRequest {
```

Comment:
The wording of the function `paymentRequest` implies a simple returning of a payment request with no side effects. What about `createPaymentRequest(with:)` instead? Sounds more substantial.

```suggestion
        internal mutating func createPaymentRequest(with supportedNetworks: [PKPaymentNetwork]) -> PKPaymentRequest {
```